script/Arcueid/Day 10/10_00_ARC10LIVING.txt

```diff
@@ -481,12 +488,12 @@ I feel a sense of dread.
 [sha:228e944ea8be70453f6eade15c57e7450c26459c]{
 -- Page 918, Offset 26926.
 -- 秋葉にはバレていなかったけど、俺の顔色、まだ悪かったのか……？
-Akiha hasn't noticed it, but is the look on my face still that bad...?
+Akiha didn't notice, but is my complexion really that bad...?
 }
 [sha:e3403d1b68394fc547a33d835024678ab8d556f7]{
 -- Page 919, Offset 26927.
 -- 「やっぱりお止めするべきですよね。
-"Perhaps I should be stopping you from going.
+"I think I should be stopping you from going.
```

Comment:
This one reads a bit strange to me

Maybe "Perhaps I should stop you from going after all."